cas/cas.c

```diff
@@ -285,6 +286,63 @@ M0_INTERNAL bool m0_crv_is_none(const struct m0_crv *crv)
 	return memcmp(crv, &M0_CRV_INIT_NONE, sizeof(*crv)) == 0;
 }
 
+M0_INTERNAL bool m0_cas_fop_is_redoable(struct m0_fop *fop)
+{
+	struct m0_fop_type *cas_fopt = &cas_put_fopt;
```

Comment:
we may need to add "del" operation here?

Reply:
@huanghua78: Agree, This can be taken up as part of integration. In this PR only basic log implementation is covered.